libfreerdp/core/proxy.c

```diff
@@ -425,7 +437,7 @@ static BOOL http_proxy_connect(BIO* bufferedBio, const char* hostname, UINT16 po
 	{
 		if (resultsize >= sizeof(recv_buf) - 1)
 		{
-			WLog_ERR(TAG, "HTTP Reply headers too long.");
+			WLog_ERR(TAG, "HTTP Reply headers too long: %s", remove_crlf_in_string(recv_buf));
```

Comment:
not really sure it is a good idea to dump whole requests/responses in production code, it might contain session keys / ...

Reply:
I don't think that the proxy response will contain any sensible data. This will only contain the HTTP response of the connect request to the proxy and we only output the response in case of an error (so there can't be any active session).

Reply:
Proposal: I could change the code so that only the first line (the status code) will be printed (i.e. replace the first occurance of \n or \r with a \0 in recv_buf and print that).